src/InverseSolve/models.jl

```diff
@@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
 	return f(m),sdiag(df(m))
 end
 
+function identityMod(m)
+	return m,1.0
```

Comment:
small thing. How about using ```UniformScaling(1.0)``` instead of 1.0? Thereby the return should be equivalent to an identity matrix.